internal/kafka/kafka.go

```diff
@@ -94,8 +94,16 @@ func NewConsumerEventLoop(
 			case kafka.Error:
 				endpoints.IncConsumeErrors()
 				l.Log.Errorf("Consumer error: %v (%v)\n", e.Code(), e)
+			case kafka.OffsetsCommitted:
+				if e.Error != nil {
+					l.Log.Errorf("Unable to commit offset: %#v", e)
+					break
+				}
+
+				l.Log.Debugf("Ignored OffsetsComitted: %#v", e)
```

Comment:
What do you think about making this a trace level log?

Reply:
Yes, even better. Done!